chef-automate-ha/nested/availabilitySetsResource.json

```diff
@@ -6,7 +6,9 @@
             "type": "object"
         }
     },
-    "variables": {},
+    "variables": {
+        "provider": "[toUpper('33194f91-eb5f-4110-827a-e95f640a9e46')]"
```

Comment:
if this is the same guid as used in the main template, why wouldn't we pass it through as a parameter instead of re-defining in each template?